test/unittests/export_plugin/benchmark_pathspec.cc

```diff
@@ -0,0 +1,74 @@
+#include <stdint.h>
+#include <sys/time.h>
+
+
+#include <ctime>
+#include <cstring>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "path_filters/dirtab.h"
+#include "export_plugin/spec_tree.h"
+
+std::string& rtrim(std::string& str, const std::string& chars = "\t\n\v\f\r ")
+{
+    str.erase(str.find_last_not_of(chars) + 1);
+    return str;
+}
+
+int main() {
+  struct timeval timeStart,
+                timeEnd;
+  long double diff = 0;
+  SpecTree *specs;
+  // catalog::Dirtab *specs;
+  gettimeofday(&timeStart, NULL);
+  specs = SpecTree::Create("/home/pteuber/Documents/cernvmfs/notes/benchmark-pathspec/cvmfs-atlas.cern.ch-bench4.trace.spec.txt");
```

Comment:
This is currently a hardcoded path. I believe this was just here as reference for performance analysis, but we may want move/rename/modify it.